packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -162,9 +200,37 @@ class MasterDetailLayout extends ResizeMixin(ElementMixin(ThemableMixin(PolylitM
         sync: true,
         observer: '__masterMinSizeChanged',
       },
+
+      /**
+       * Define how master and detail areas are shown next to each other,
+       * and the way how size and min-size properties are applied to them.
+       * Possible values are: `horizontal` or `vertical`.
+       * Defaults to horizontal.
+       */
+      orientation: {
+        type: String,
+        value: 'horizontal',
+        reflectToAttribute: true,
+        observer: '__orientationChanged',
+        sync: true,
+      },
     };
   }
 
+  constructor() {
+    super();
+
+    this.__onWindowResize = this.__onWindowResize.bind(this);
+  }
+
+  /**
+   * @return {boolean}
+   * @protected
+   */
+  get _vertical() {
```

Comment:
Nit: I'd make everything private by default, unless we get a use case where something needs to become public API.

Reply:
This getter is actually not needed so I removed it altogether.